region/wreckedship/main/Bowling Alley.json

```diff
@@ -727,15 +752,43 @@
       "name": "Crystal Flash Grapple Clip",
       "requires": [
         "h_canJumpIntoCrystalFlashClip",
-        "Grapple"
+        "Grapple",
+        {"obstaclesNotCleared": ["B"]}
       ],
       "note": "Menu to Grappling Beam before the crystal flash ends and mash shoot while holding down."
     },
+    {
+      "link": [5, 1],
+      "name": "Damage Down then Crystal Flash Grapple Clip",
+      "requires": [
+        {"obstaclesCleared": ["B"]},
+        {"or": [
+          "h_ShinesparksCostEnergy",
+          "f_DefeatedPhantoon"
```

Comment:
The `f_DefeatedPhantoon` case may run into a problem if you have exactly 2 ETanks (or 5, 8, 11, or 14). In this case after the refill to 299 energy, you would be able to damage down to 59 energy, but then the next spike hit would kill you. If you have a suit then it should be ok, assuming suits cut spike damage by at least half; maybe this should be a helper as well? Also if you can reach the Workrobots (by destroying the Chozo PB blocks) then they could maybe be used to damage down in smaller increments, but I haven't tested if it actually works here (e.g. while off-camera).

Reply:
Looks like they can shoot you while off camera very slowly (but worst case scenario of 59 energy, you only need to take 3 hits). You need PBs to get to them though.
Alternatively, any amount of reserves + the spikes are enough.

Reply:
suits dont need to protect you by at least half, as long as its some protection and not full protection, you can toggle a suit for some of the spike hits and be fine. 

I dont really think it needs to be a helper, since wouldnt basically all suit possibilities have some and not full protection?

Reply:
Looks like with VARIA randomizer "Balanced" suit option, Gravity provides no protection against spikes

Reply:
Yes, not providing spike damage reduction stands out and was the default for most randomizers.  But then Varia was still global 50%.

I took it out.  I don't want to lock the camera to bottom left when trying to jumpintoCF by visiting the robots.  But shinespark damage is still fine as a way to damage down.